scripts/run_performance_tests.sh

```diff
@@ -86,7 +96,18 @@ else
   export XVFB_PREFIX="/usr/bin/xvfb-run"
 fi
 
-$XVFB_PREFIX $PYTHON_CMD scripts/backend_tests.py --test_path='core/tests/performance_tests' $@ 
+if [ $# -eq 0 ]; then
```

Comment:
The `$#` variable gives you the number of input arguments the script was passed.